typescript/vrotsc/src/types.ts

```diff
@@ -69,6 +83,7 @@ export interface FileTransformationContext {
 	outputs: ProgramOutputs;
 	diagnostics: DiagnosticCollection;
 	sourceFiles: ts.SourceFile[];
+	configIdsMap: any;											// Holds ConfigElement Path -> Id mappings.
```

Comment:
Same as type definitions i other file.
It would be better to specify exactly that the type is dictionary of key: string, any/string(value).
{ [key: string]: any } or { [key: string]: string }

Reply:
Done